perf/CMakeLists.txt

```diff
@@ -5,6 +5,16 @@ file(MAKE_DIRECTORY ${PERF_OUTPUT_DIR})
 
```

Comment:
don't forget to update a dir name in commit message, see a comment from Alexander in a previous commit

Reply:
Updated, thanks!